src/NServiceBus.Core.Tests/Unicast/MessagingBestPracticesTests.cs

```diff
@@ -50,7 +50,6 @@ public void Should_throw_for_command()
             public void Should_not_throw_for_event()
             {
                 new Validations(new Conventions()).AssertIsValidForPubSub(typeof(MyEvent));
-                //TODO: verify log
```

Comment:
Couldn't even remember what that meant. Is there enough ROI to check the logs here?